lib/devise/multi_email/models/authenticatable.rb

```diff
@@ -69,27 +52,18 @@ def find_first_by_auth_conditions(tainted_conditions, opts = {})
 
           if email && email.is_a?(String)
             conditions = filtered_conditions.to_h.merge(opts).
-              reverse_merge(emails: { email: email })
+              reverse_merge(multi_email_association.reflection.table_name => { email: email })
```

Comment:
@allenwq I can change it back to use just the association name, and see if there are issues, but hard to test all scenarios possibly.